package.json

```diff
@@ -41,14 +44,18 @@
     "@nestjs/cli": "^8.0.0",
     "@nestjs/schematics": "^8.0.0",
     "@nestjs/testing": "^8.0.0",
```

Comment:
@andersonaguiar @xenide How do we feel about not using `^` and instead locking in specific versions to make our deps a little safer at the cost of auto updates? I have a preference towards stability but might not be appropriate for this repo

Reply:
having the `package-lock.json` will mitigate the problem, but I'm fine with that if everyone agree

Reply:
how often do you use `npm ci` vs `npm i`/`npm install`?

Reply:
I never use `npm ci`, just use `npm i`. Sometimes I remove the `node_modules` folder but I never delete/override the `package-lock.json` in this way it will be always in sync with everyone.

Reply:
`npm i` updated `package-lock.json` with latest deps allowing new versions to make it into your codebase passively (as opposed to explicitly updating the `package.json`)

Reply:
Btw, if you don't auto bump you will still want to update periodically. For this I recommend:
 - `npm outdated`
 - `npm install -g npm-check-updates && ncu -u` / `npx ncu -u`

Reply:
maybe we can just lock the versions in the `package.json` to be safer

Reply:
If you're on OS / Linux I recommend:
```bash
echo "save-exact=true" >> ~/.npmrc
```

This will make each `npm install __dep__` save the exact version (`vX.X`) as opposed to the caret version (`^vX.X`)

Reply:
So what do we want to do for this? Lock the versions in package.json? Along with `save-exact` in `.npmrc` ?

Reply:
I would be for this, even though it is not a bad idea to stay up to date on patches for an API, I think this is cleaner. So to confirm:
```
1. Add `save-exact` to `.npmrc`
2. Replace all `^` with ``
```